README.md

```diff
@@ -18,16 +18,22 @@ The following instruction assume your are using Python 2 and an Ubuntu installat
 
 ## Dependencies
 
-Pymavlink requires two dependencies :
+Pymavlink requires 3 dependencies :
 
-    - future : for python 2 and python 3 interoperability
-    - lxml : for checking and parsing xml file
+    - future : for python 2 and python 3 interoperability (http://python-future.org/)
+    - lxml : for checking and parsing xml file (http://lxml.de/installation.html)
 
 ### On linux
 
-Using pip you can install the required dependencies
+Lxml require some dependencies too
```

Comment:
capitalisation, and should be "requires", not "require".

Probably better to be "lxml has some additional dependencies"